### PR TITLE
Units test to investigate AMQP generation 

### PR DESCRIPTION
**Pull request by pdalfarr**
Units test to investigate AMQP generation
I added some comments in the PR.

### CODE REVIEW

File: springwolf-examples/springwolf-amqp-example/src/main/java/io/github/springwolf/examples/amqp/AmqpConstants.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+package io.github.springwolf.examples.amqp;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class AmqpConstants {
```
Comment (pdalfarr) [inline springwolf-examples/springwolf-amqp-example/src/main/java/io/github/springwolf/examples/amqp/AmqpConstants.java:8]: Add constant to ease navigation in code

File: springwolf-examples/springwolf-amqp-example/src/main/java/io/github/springwolf/examples/amqp/consumers/ExampleConsumer.java [hunk 2]
```diff
@@ -31,37 +35,106 @@ public void receiveExamplePayload(ExamplePayloadDto payload) {
...
+
+    @RabbitListener(
+            autoStartup = "false",
+            bindings =
+                    @QueueBinding(
+                            exchange =
+                                    @Exchange(
+                                            name = AmqpConstants.EXCHANGE_CRUD_TOPIC_EXCHANGE_1,
+                                            type = ExchangeTypes.TOPIC),
+                            key = AmqpConstants.ROUTING_KEY_ALL_MESSAGES,
```
Comment (pdalfarr) [inline springwolf-examples/springwolf-amqp-example/src/main/java/io/github/springwolf/examples/amqp/consumers/ExampleConsumer.java:113]: using "#" routing key

**Comment by timonback** [comment]
Hi @pdalfarr,
I took this PR as a base to look into amqp and further understand how rabbitmq and the AsyncAPI amqp binding works. The result is https://github.com/springwolf/springwolf-core/pull/886

I agree with your point on the expected channel name.
Assuming that is fixed, how to bind the routingKey + exchange with the queue (see my comment)?

Also, it would be great if you can verify the comments on RabbitListenerUtil as I gather them by testing, but have low confidence since I haven't used RabbitMQ in production (see https://github.com/springwolf/springwolf-core/pull/886/files#r1699142615 )

**Comment by timonback** [comment]
Closing this PR, as the proposed changes have been merged via other PRs.

Also, part of the issue has been addressed, while the link from exchange to queue is missing in the AsyncAPI at this point. Lets move the discussion into the original Springwolf issue https://github.com/springwolf/springwolf-core/issues/366

**Pull request closed without merging**